packages/plugin-core/package.json

```diff
@@ -225,6 +225,10 @@
         "command": "dendron.randomNote",
         "title": "Dendron: Random Note"
       },
+      {
```

Comment:
unrelated to this review, but I think we don't actually need this command to be registered? Can we remove from const.ts (maybe separate review).